browse/static/js/catalyzex.js

```diff
@@ -0,0 +1,59 @@
+(async () => {
+  const arxivId = window.location.pathname.split("/").reverse()[0];
```

Comment:
To account for URLs like https://arxiv.org/abs/1812.04948 as well as https://arxiv.org/abs/cs/0609047 (whose arxiv id would be `cs/0609047`

```suggestion
  const arxivId = window.location.pathname.match(/(?:abs|pdf)\/(.+\d{3})/)[1];
```

Reply:
Done :)